aten/src/ATen/native/cuda/BinaryOpsKernel.cu

```diff
@@ -13,7 +13,7 @@
 namespace at { namespace native {
 
 void add_kernel_cuda(TensorIterator& iter, Scalar alpha_scalar) {
-  AT_DISPATCH_ALL_TYPES_AND2(kHalf, kBool, iter.dtype(), "add_cuda", [&]() {
+  AT_DISPATCH_ALL_TYPES_AND2(kHalf, kBool, iter.dtype(), "add_cuda/sub_cuda", [&]() {
```

Comment:
I wouldn't change this

Reply:
Are you sure about this? An error about "add_cuda" may pop up (instead of "sub_cuda") when the user uses `-`.

Reply:
In which case will that happen? im trying out your change and in cases `1 - boolTensor` and `boolTensor - boolTensor` im getting the error that you are providing: ""Subtraction, the `-` operator, on bool ..."

Reply:
The point is, as done in the cpu version (where half is not supported), this is preventive of any unsupported future dtype causing this issue. IMO, there is no harm to make the change, although the benefit is uncertain.

Reply:
ok, i guess it makes sense.